Examples/Kinova/Armour/KinovaSuctionCup.yaml

```diff
@@ -56,5 +56,31 @@ KinovaWithSuctionCup:
       "radius": 0.02
     - "offset": [0.03, -0.06, -0.055]
       "radius": 0.02
+  tapered_capsules_1:
+    joint_2:
+      - "sphere_1": "collision-0"
+      - "sphere_2": "collision-6"
+    joint_4:
+      - "sphere_1": "collision-7"
+      - "sphere_2": "collision-11"
+    joint_6:
+      - "sphere_1": "collision-12"
+      - "sphere_2": "collision-13"
+    joint_7:
+      - "sphere_1": "collision-14"
+      - "sphere_2": "collision-16"
+  tapered_capsules_2:
```

Comment:
I'm not sure how to define things for two arms yet. For now let's just worry about self-intersection constraints.